Update from 1.25.0 to 1.26.0 causes NoClassDefFoundError on 'Preconditions'

#### Environment details

- OS: OSX 10.12.6
- Java version:  1.8.0_111-b14, 64 bit
- google-http-java-client version: 1.25.0/1.26.0

#### Steps to reproduce

1. The Statement ```Preconditions.checkState(true);``` works fine in 1.25.0.
2. Then switch to Version 1.26.0.
3. Now you will get a ```NoClassDefFoundError```
4. Then add guava (version 20.0) and everything is good again. 

#### Stacktrace

```
java.lang.NoClassDefFoundError: com/google/common/base/Preconditions

	at com.google.api.client.util.Preconditions.checkState(Preconditions.java:81)
	at org.soundtouch4j.ZoneApiTest.testPrecondition(ZoneApiTest.java:172)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
Caused by: java.lang.ClassNotFoundException: com.google.common.base.Preconditions
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 41 more
```

#### Any additional information below

Making sure to follow these steps will guarantee the quickest resolution possible.

In the class ```Preconditions``` there is a change from ```com.google.api.client.repackaged.com.google.common.base.Preconditions.checkArgument(expression);``` to ```com.google.common.base.Preconditions.checkArgument(expression); ```. And this package is not part of the JAR File.

--- a/google-http-client/pom.xml
+++ b/google-http-client/pom.xml
@@ -142,7 +142,6 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.google.guava</groupId>

